solanches/rest.py

```diff
@@ -166,17 +166,14 @@ def _error(error):
 @app.route("/comercio/<comercio_nome>", methods=['PATCH'])
 def update_comercio(comercio_nome):
 
-    req = request.get_json()  
-    assert req, "Erro: json inválido!"
+    attributes = request.get_json()  
+    assert attributes, "Erro: json inválido!"
```

Comment:
O que você recebe no request.get_json é o json da requisição, então deve ficar como estava, `req = request.get_json()`. Minha observação se referia ao conteúdo que é repassado pro controller que continuou o mesmo, já que tu só fez renomear a variável. Ao invés de passar o `req` pro controller, você deve capturar o conteúdo do campo `"attributes"` e repassá-lo:

```suggestion
    req = request.get_json()  
    assert req, "Erro: json inválido!"
    
    attributes = req.get("attributes", {})
```